README.md

```diff
@@ -1 +1,13 @@
-# angular-testing-custom-filters-lab
\ No newline at end of file
+# Testing filters with Jasmine/Karma
+
+## Objectives
+
+- Write unit tests for our custom filters
+
+## Instructions
+
+In this repo we've got two filters.
+
+The first filter is gender - it will filter an array of objects depending on the `gender` property in the object. Test that this filter is filtering a mocked example list of people correctly.
+
+The second is a filter to remove all the vowels in a string. Test that this removes every vowel in any string given to the filter.
```

Comment:
Add an instruction to run the specs once they've written them to make sure that they pass!